src/controllers/check.your.answers.controller.ts

```diff
@@ -18,9 +20,21 @@ export const get = (req: Request, res: Response, next: NextFunction) => {
 
     const appData: ApplicationData = getApplicationData(req.session);
 
+    let hasTrusts: boolean = false;
+
+    if (isActiveFeature(config.FEATURE_FLAG_ENABLE_TRUST_INFO_16062022)) {
+      hasTrusts = checkEntityHasTrusts(appData);
+    }
+
+    let backLinkUrl: string = config.BENEFICIAL_OWNER_TYPE_URL;
+    if (hasTrusts) {
+      backLinkUrl = config.TRUST_INFO_PAGE;
+    }
+
     return res.render(config.CHECK_YOUR_ANSWERS_PAGE, {
-      backLinkUrl: config.BENEFICIAL_OWNER_TYPE_URL,
+      backLinkUrl: backLinkUrl,
```

Comment:
if the param name is same as variable name eg backLinkUrl: backLinkUrl, you can just use backLinkUrl on it's own, same for hasTrusts  eg
 return res.render(config.CHECK_YOUR_ANSWERS_PAGE, {
      backLinkUrl,
      templateName: config.CHECK_YOUR_ANSWERS_PAGE,
      hasTrusts,
      appData
    }